src/DefaultBuilder/test/Microsoft.AspNetCore.Tests/StaticWebAssets/StaticWebAssetsFileProviderTests.cs

```diff
@@ -0,0 +1,72 @@
+// Copyright (c) .NET Foundation. All rights reserved.
+// Licensed under the Apache License, Version 2.0. See License.txt in the project root for license information.
+
+using System;
+using System.IO;
+using System.Runtime.InteropServices;
+using Xunit;
+
+namespace Microsoft.AspNetCore.Tests.StaticWebAssets
+{
+    public class StaticWebAssetsFileProviderTests
+    {
+        [Fact]
+        public void StaticWebAssetsFileProvider_ConstructorThrows_WhenPathIsNotFound()
+        {
+            // Arrange, Act & Assert
+            var provider = Assert.Throws<DirectoryNotFoundException>(() => new StaticWebAssetsFileProvider("/prefix", "/nonexisting"));
+        }
+
+        [Fact]
+        public void StaticWebAssetsFileProvider_Constructor_PrependsPrefixWithSlashIfMissing()
+        {
+            // Arrange & Act
+            var provider = new StaticWebAssetsFileProvider("_content", AppContext.BaseDirectory);
+
+            // Assert
+            Assert.Equal("/_content", provider.BasePath);
```

Comment:
I don't really have an context about how to write these tests in a way that's friendly to helix, but it seems worthy to figure out. (if you haven't already).